app/models/profile.rb

```diff
@@ -25,9 +25,9 @@ def decrypt_pii(user_access_key)
   end
 
   def recover_pii(recovery_code)
-    raise 'invalid recovery code' unless recovery_code_generator.verify(recovery_code)
-    uak = recovery_code_generator.user_access_key
-    Pii::Attributes.new_from_encrypted(encrypted_pii_recovery, uak)
+    rc_user_access_key = UserAccessKey.new(recovery_code, ssn_signature)
```

Comment:
Would it just be easier to add a second salt column to the user? ex `password_salt` and `recovery_code_salt`? Would we ever need to recover data without an SSN?